app/helpers/role_organisations_helper.rb

```diff
@@ -1,6 +1,6 @@
 module RoleOrganisationsHelper
   def options_for_your_organisation_select(current_user)
-    organisations = Pundit.policy_scope(current_user, Organisation).order(:name)
+    organisations = Pundit.policy_scope(current_user, Organisation).order(:name).where(closed: false)
```

Comment:
Subjective: I'd expect calls to `#order` to go after calls to `#where`.

Reply:
Subjective: It might be nice to extract an `Organisation.not_closed` scope to make this slightly more readable.

Reply:
Fixed the method chain order in [09e0e04](https://github.com/alphagov/signon/pull/2417/commits/09e0e04297394e8d1d027c7750eff9a6c17f8cc5)

Reply:
Introduced the new scope in 35b9d0be